Running "module reset". Resetting modules to system default. The following $MODULEPATH directories have been removed: None
2025-04-09 02:01:40 | INFO | fairseq_cli.eval_lm | Namespace(add_bos_token=False, all_gather_list_size=16384, bpe=None, context_window=2560, cpu=False, criterion='cross_entropy', data='data-bin/wikitext-103', dataset_impl=None, dstore_filename='cdatastore/wikitext-103/train', dstore_fp16=False, dstore_mmap=None, dstore_size=103225485, empty_cache_freq=0, faiss_metric_type='l2', force_anneal=None, fp16=True, fp16_init_scale=128, fp16_no_flatten_grads=False, fp16_scale_tolerance=0.0, fp16_scale_window=None, future_target=False, gen_subset='valid', indexfile='cdatastore/wikitext-103/train/knn.index', k=1024, knn_keytype='last_ffn_input', knn_sim_func=None, knnlm=True, lm_eval=True, lmbda=0.25, log_format=None, log_interval=1000, lr_scheduler='fixed', lr_shrink=0.1, max_sentences=None, max_target_positions=None, max_tokens=3072, memory_efficient_fp16=False, min_loss_scale=0.0001, model_overrides="{'knn_keytype': 'last_ffn_input'}", momentum=0.99, move_dstore_to_mem=False, no_load_keys=False, no_min_context=False, no_progress_bar=False, num_shards=1, num_workers=1, optimizer='nag', output_dictionary_size=-1, output_word_probs=False, output_word_stats=False, past_target=False, path='adaptive_lm_wiki103.v2/model.pt', probe=32, quiet=False, remove_bpe=None, required_batch_size_multiple=8, results_path=None, sample_break_mode='complete', save_knnlm_dstore=False, seed=1, self_target=False, shard_id=0, skip_invalid_size_inputs_valid_test=False, softmax_batch=1024, task='language_modeling', tensorboard_logdir='', threshold_loss_scale=None, tokenizer=None, tokens_per_sample=1024, truncate_sequence=False, user_dir=None, warmup_updates=0, weight_decay=0.0)
2025-04-09 02:01:40 | INFO | fairseq.tasks.language_modeling | dictionary: 267744 types
2025-04-09 02:01:40 | INFO | fairseq_cli.eval_lm | loading model(s) from adaptive_lm_wiki103.v2/model.pt
2025-04-09 02:01:50 | INFO | fairseq.tasks.language_modeling | dictionary: 267744 types
2025-04-09 02:01:50 | INFO | fairseq.data.data_utils | loaded 3760 examples from: data-bin/wikitext-103/valid
2025-04-09 02:01:51 | INFO | fairseq_cli.eval_lm | data-bin/wikitext-103 valid 503 examples
2025-04-09 02:04:26 | INFO | fairseq_cli.eval_lm | num. model params: 246933504
Traceback (most recent call last):
  File "eval_lm.py", line 11, in <module>
    cli_main()
  File "/work/hdd/bcyi/eergun/knnlm-retrieval-quality/fairseq_cli/eval_lm.py", line 309, in cli_main
    main(args)
  File "/work/hdd/bcyi/eergun/knnlm-retrieval-quality/fairseq_cli/eval_lm.py", line 153, in main
    knn_dstore = KNN_Dstore(args)
  File "/work/hdd/bcyi/eergun/knnlm-retrieval-quality/fairseq/knnlm.py", line 18, in __init__
    self.index = self.setup_faiss(args)
  File "/work/hdd/bcyi/eergun/knnlm-retrieval-quality/fairseq/knnlm.py", line 26, in setup_faiss
    index = faiss.read_index(args.indexfile, faiss.IO_FLAG_ONDISK_SAME_DIR)
  File "/scratch/bcyi/esenergun/knnlm/lib/python3.8/site-packages/faiss/swigfaiss_avx2.py", line 10538, in read_index
    return _swigfaiss_avx2.read_index(*args)
RuntimeError: Error in faiss::FileIOReader::FileIOReader(const char*) at /project/faiss/faiss/impl/io.cpp:67: Error: 'f' failed: could not open cdatastore/wikitext-103/train/knn.index for reading: No such file or directory
